ui/src/testing/stubs/workspaces-api-stub.ts

```diff
@@ -50,9 +48,7 @@ export class WorkspacesApiStub extends WorkspacesApi {
   newWorkspaceOperationCount = 0;
 
   constructor(workspaces?: Workspace[], workspaceUserRoles?: UserRole[]) {
-    super(undefined, undefined, (..._: any[]) => {
-      throw stubNotImplementedError;
-    });
+    super(undefined);
```

Comment:
This ultimately extends BaseApi which only takes one parameter: a configuration object.